lib/rules/require-toggle-inside-transition.js

```diff
@@ -30,14 +30,31 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/require-toggle-inside-transition.html'
     },
     fixable: null,
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          additionalDirectives: {
+            type: 'array',
+            items: {
+              type: 'string'
+            }
+          }
+        },
+        additionalProperties: false
+      }
+    ],
     messages: {
       expected:
         'The element inside `<transition>` is expected to have a `v-if` or `v-show` directive.'
```

Comment:
Should we also update the message to display all allowed directives?

Reply:
Yes, I think that would make sense.

Reply:
This sounds good to me as well. Added to the PR